### ISSUE TITLE
Cross-repo find implementations for remote type and remote interface

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/scip-go#65]
Similar to https://github.com/sourcegraph/scip-go/issues/64 but for the case for when both the type and interface do not belong to the current repo.

**Comment by jupblb** [comment]
I don't think this is feasible on SCIP indexer side. But working on https://github.com/sourcegraph/scip-go/issues/64 should make it possible.
